### DISCUSSION TITLE
Will browser mode be included in the @vitest-environment?

### DISCUSSION THREAD

**Discussion by 70-10** [vitest-dev/vitest#4151] [Q&A] [answered]
I want to select browser mode like `@vitest-environment`

```
// @vitest-environment browser:chrome
.
.
.
```

In other words, depending on the test, I want to run some tests in browser mode and others in non-browser mode.  
Is there any concept of a browser mode that could be set for each test?

thanks.

**Comment by sheremet-va** [comment, marked as answer]
There are no plans for this. There will probably more configuration available in the config tho.

**Comment by AmirTugi** [comment]
I find this feature relevant to me, as there are some tests that require painting the component on screen, and the others are regular unit-tests.

I don't wish to hurt performance for the regular unit-tests for a few tests that require browser.

Perhaps we can take advantage of the [vitest workspace](https://vitest.dev/guide/workspace.html) to make 2 different sets of tests.
This might also help to run the browser and regular unit-test in parallel.
